### PR TITLE
[New Designs] Update OONI Run v1 UI to v2

### PR DESCRIPTION
**Pull request by aanorbel**
Fixes  https://github.com/ooni/probe/issues/2596

## Proposed Changes

  - Update designs for Run v1 components to match new designs
  - Add `SrtingListRecyclerViewAdapter` to display List items
  - Convert `OoniRunActivity` to Kotlin
  
| Light | Dark|
|--|--|
|.|.|
|web_connectivity_light|web_connectivity_dark|
|web_connectivity_light_long_list| web_connectivity_dark_long_list|
|web_connectivity_light_short_list|web_connectivity_dark_short_list|
|http_header_field_manipulation_light|http_header_field_manipulation_dark|
|invalid_light|invalid_dark|
|outdated_light|outdated_dark|
|whatsapp_light|whatsapp_dark?

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -108,7 +108,7 @@
 		</activity>
 		<activity
 			android:name=".activity.OoniRunActivity"
-			android:label="@string/APP_NAME"
+			android:label=""
```
Comment (bassosimone) [inline app/src/main/AndroidManifest.xml:111]: I would like to know why we don't need this label anymore

File: app/src/main/java/org/openobservatory/ooniprobe/adapters/StringListRecyclerViewAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        return ViewHolder(
+            ItemRunV1Binding.inflate(
+                LayoutInflater.from(parent.context), parent, false
+            )
+        )
+    }
+
+    /**
+     * Replaces the contents of a view (invoked by the layout manager).
+     */
```
Comment (bassosimone) [inline app/src/main/java/org/openobservatory/ooniprobe/adapters/StringListRecyclerViewAdapter.kt:28]: I like that you added this kind of comments 🙏 💯

**Review by bassosimone — CHANGES_REQUESTED** [review]
🐳 

Thanks for modernizing the codebase!

We should make sure we address the merge conflict before merging: it seems something changed in the original OONI Run v1 class and we should make sure to carry forward what changed in the new kotlin code!

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (aanorbel) [reply app/src/main/AndroidManifest.xml:111]: This displays at the title of the page. Adding it will mean we can no longer conform to the designs

**Pull request merged**